tests/conftest.py

```diff
@@ -4,34 +4,52 @@
 import requests_mock
 from sdk.device.device_types import DeviceType
 
+from sdk.endpoints import Endpoints
 
 TEST_API_FIXTURES_PATH = "tests/fixtures/api"
 JSON_FILE = "_.{}.json"
 
 
+def mock_core_response(request):
+    path = request.url.split("/api/v1/")[1].split("?")[0]
+    
+    data = None
+    if request.method == "POST":
+        data = request.json()
+
+    json_path = os.path.join(
+        TEST_API_FIXTURES_PATH, path, JSON_FILE.format(request.method)
+    )
+    with open(json_path) as json_file:
+        result = json.load(json_file)
+        if data:
+            if data.get("emulator"):
+                result["data"]["device_type"] = data["emulator"]
+            else:
+                result["data"]["device_type"] = DeviceType.QPU
+                result["data"]["configuration"] = None
+        return result
+
+
+def mock_auth0_response():
+    json_path = "tests/fixtures/auth0/token_response.json"
+    with open(json_path) as json_file:
+        result = json.load(json_file)
+
+    return result
+
+
+def mock_response(request, context):
+    if request.url.startswith(Endpoints.core):
```

Comment:
I see you are not mocking the calls to account. Does that mean the SDK isn't using it anymore now that we got rid of the login directly ? We can maybe remove it from the `Endpoints` class then!

Reply:
I deleted the Endpoints to Account, but I have modified it because it's used in the `self.audience` of the authentication.
I think we may want to develop a more simple system for the sdk that will change the baseurl based on the `local | dev | preprod | prod` environment. But it wasn't in the task I was assigned :)